src/components/AddItemForm.tsx

```diff
@@ -1,22 +1,38 @@
-import React from 'react';
+import React, { useState } from 'react';
 
-function AddItemForm() {
+function AddItemForm(props: any) {
```

Comment:
Não podemos usar o type `any`, senao é o mesmo de não usar typescript.

Recomendo:
- criar uma interface para suas Props
- dar o spread(abertura) dos argumentos na assinatura da função

```suggestion
interface Props {
  // sua implementação aqui
}

function AddItemForm({ setDescription, description }: Props) {
```

Links para estudo:
- https://www.typescriptlang.org/docs/handbook/interfaces.html
- https://www.pluralsight.com/guides/defining-props-in-react-function-component-with-typescript